pkgs/checks/test/extensions/core_test.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:checks/checks.dart';
```

Comment:
[nit] copyright

Reply:
Uh...you don't have them anywhere else in this package. Follow-up with a one-off to add em?